### PR TITLE
Fix typo on UicFacility  enumeration value allinclusivePrice

### PR DESCRIPTION
**Pull request by nick-knowles**
corerctfrom AllIInclusivePrice AllInclusivePrice

UicServiceFacility value -

 \netex_facilityUic_support.xsd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,18 +30,100 @@ There are **XML examples** of the use of both protocols, see */examples* subdire
...
   - NeTEx.spp
 
-There is also an _Oxygen_  project file:
+There is also an _Oxygen_ project file:
   - NeTEx.xpr
 ----
 # Change Log
 
-### 2019.05.17 FIX  *PART3:FARES*: NJSK  __FarePointInPattern__Fix case on __isFareStage__ and __isForbidden__  
+### 2020.08.12   FIX  Typo on _AllInclusivePrice_	to  __UicProductCharacteristics__  /  ___
```
Comment (skinkie) [inline README.md:38]: I doubt this all is in the change request.

**Review by ue71603 — CHANGES_REQUESTED** [review]
I suggest to omit the README.md and just to the PR for the other file. Otherwise we merge an old README.md into the new one.

**Review by Aurige — APPROVED** [review]

**Comment by skinkie** [comment]
@Aurige the other pull request... #320

**Comment by Aurige** [comment]
I only see the README.md in 320 not the typo correction itself

**Comment by skinkie** [comment]
> I only see the README.md in 320 not the typo correction itself

https://github.com/NeTEx-CEN/NeTEx/pull/320/files

**Pull request closed without merging**